src/dialogs/voice-command-dialog/show-ha-voice-command-dialog.ts

```diff
@@ -1,8 +1,18 @@
 import { fireEvent } from "../../common/dom/fire_event";
+import { HomeAssistant } from "../../types";
 
 const loadVoiceCommandDialog = () => import("./ha-voice-command-dialog");
 
-export const showVoiceCommandDialog = (element: HTMLElement): void => {
+export const showVoiceCommandDialog = (
+  element: HTMLElement,
+  hass: HomeAssistant
```

Comment:
Should we expose the options here.

Reply:
I guess we should do that in a follow-up PR, as it would also need to be added to `ha-voice-command-dialog`.